chapter_appendix_math/single-variable-calculus_vn.md

```diff
@@ -339,14 +339,14 @@ and see that as we send $\epsilon \rightarrow 0$, the right hand term goes to ze
 Finally, with the chain rule, we can again progress as before using :eqref:`eq_small_change` and see that
 -->
 
-*dịch đoạn phía trên*
+Cuối cùng, theo quy tắc đạo hàm của hàm số hợp, chúng ta có thể tiếp tục khai triển như lúc trước, sử dụng :eqref:`eq_small_change` và thấy rằng:
 
 $$
 \begin{aligned}
 f(x+\epsilon) & = g(h(x+\epsilon)) \\
 & \approx g\left(h(x) + \epsilon \frac{dh}{dx}(x)\right) \\
 & \approx g(h(x)) + \epsilon \frac{dh}{dx}(x) \frac{dg}{dh}(h(x))\\
-& = f(x) + \epsilon \frac{dg}{dh}(h(x))\frac{dh}{dx}(x),
+& = f(x) + \epsilon \frac{dg}{dh}(h(x))\frac{dh}{dx}(x).
```

Comment:
```suggestion
& = f(x) + \epsilon \frac{dg}{dh}(h(x))\frac{dh}{dx}(x),
```

Chỗ này tác giả chủ ý dùng ',' vì vẫn chưa hết "câu". Ở dưới còn "where" viết thường nữa.

Reply:
Ah, tại chữ `where` khó dịch trôi quá, tui ngắt câu luôn, nên đổi thành dấu `.`. :)

Reply:
@tiepvupsu chỗ này có ok to resolve ko? Bạn có gợi ý nào để dịch chữ `where` ở dưới sát hơn ko? Thanks!